### PR TITLE
fix for over/under colors

### PR DESCRIPTION
**Pull request by ChristianBeilschmidt**

### CODE REVIEW

**Comment by ChristianBeilschmidt** [comment]
[Deploybare Version des EBV-Analyzers erstellen](https://trello.com/c/rCZOfj4w/1169-deploybare-version-des-ebv-analyzers-erstellen)

File: projects/core/src/lib/colors/colorizer.model.ts [hunk 1]
```diff
@@ -104,19 +123,23 @@ export class LinearGradient extends Colorizer {
...
         readonly noDataColor?: Color;
-        readonly defaultColor?: Color;
+        readonly overColor?: Color;
+        readonly underColor?: Color;
     }): LinearGradient {
         return new LinearGradient(
             updates.breakpoints ?? this.breakpoints.map((b) => b.clone()),
             updates.noDataColor ?? this.noDataColor.clone(),
-            updates.defaultColor ?? this.defaultColor.clone(),
+            // TODO: refactor
```
Comment (michaelmattig) [inline projects/core/src/lib/colors/colorizer.model.ts:140]: what is there to refactor?

Comment (ChristianBeilschmidt) [reply projects/core/src/lib/colors/colorizer.model.ts]: I already did this, the comment was a leftover:
[7b314a3](https://github.com/geo-engine/geoengine-ui/pull/573/commits/7b314a32ae65a87bca9de8d9427bdb979eddd26d)

**Review by michaelmattig — APPROVED** [review]

**Comment by michaelmattig** [comment]
bors merge

**Pull request merged**